app/scripts/controllers/mobileClients.js

```diff
@@ -57,6 +57,24 @@ angular.module('openshiftConsole')
                 }
                 ctrl.mobileClient = mobileClient;
               }));
+
+              watches.push(DataService.watch(APIService.getPreferredVersion("serviceinstances"), {namespace: ctrl.projectName}, function(serviceInstances){
```

Comment:
I think creating a higher level component that handles these watches and lists the services like we did for integrations might be good. It will keep the tab logic separate from the view controller.